pulp_rpm/app/models.py

```diff
@@ -501,6 +502,7 @@ class UpdateCollection(Model):
 
     name = models.TextField()
     shortname = models.TextField()
+    module = models.TextField(default='')
```

Comment:
What about serializer? Shouldn't it be updated as well?

Reply:
serializer using custom fields where it is managed ok